apps/frontend/src/pages/projects/components/project-card/project-card.tsx

```diff
@@ -2,23 +2,33 @@ import { NavLink } from "react-router-dom";
 
 import { AppRoute } from "~/libs/enums/enums.js";
 import { configureString } from "~/libs/helpers/helpers.js";
+import { useCallback } from "~/libs/hooks/hooks.js";
 import { type ProjectGetAllItemResponseDto } from "~/modules/projects/projects.js";
 
+import { ProjectMenu } from "../components.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	onEditClick: (project: ProjectGetAllItemResponseDto) => void;
 	project: ProjectGetAllItemResponseDto;
 };
 
-const ProjectCard = ({ project }: Properties): JSX.Element => {
+const ProjectCard = ({ onEditClick, project }: Properties): JSX.Element => {
 	const projectRoute = configureString(AppRoute.PROJECT, {
 		id: project.id.toString(),
 	});
 
+	const handleEditClick = useCallback(() => {
+		onEditClick(project);
+	}, [onEditClick, project]);
+
 	return (
-		<NavLink className={styles["project"] ?? ""} to={projectRoute}>
-			{project.name}
-		</NavLink>
+		<div className={styles["project"]}>
+			<NavLink className={styles["project-link"] ?? ""} to={projectRoute}>
+				{project.name}
+			</NavLink>
+			<ProjectMenu onEditClick={handleEditClick} />
+		</div>
```

Comment:
When you click on the options button, it redirects you to some project page. I was trying to fix it using event.preventDefault and event.stopPropagation, but it looks kinda messy, is there any other ways to trigger button?

Reply:
Hm, actually I'm thinking that we cannot nest a button inside link, so it should've gone underneath as previously anyway. Usually, for such cases underneath link just has absolute positioning and is stretched to cover container sizes. I think this will be a more correct way to implement this and it should fix the problem